src/JustSaying/Messaging/MessageSerialization/JustSayingJsonSerializerOptions.cs

```diff
@@ -0,0 +1,10 @@
+﻿#if NET8_0_OR_GREATER
+using System.Text.Json;
+
+namespace JustSaying.Messaging.MessageSerialization;
+
+public class JustSayingJsonSerializerOptions
+{
+    public JsonSerializerOptions SerializerOptions { get; } = new();
```

Comment:
`JsonSerializerOptions.Default` is annotated with `RequiresUnreferencedCode` and `RequiresDynamicCode`. It would also not be configurable after it's first use application-wide.